### ISSUE TITLE
1.21 Support

### ISSUE THREAD

**Issue by JayanAXHF** [maruohon/litematica#857]
When will the mod be supported for 1.21

**Comment by Mimibienv** [comment]
yes pls

**Comment by vacla** [comment]
You can find it here: https://github.com/sakura-ryoko/litematica/releases

**Comment by progsa98** [comment]
Does anyone else get an error when downloading the newest version that it's a virus?

**Comment by Samalando** [comment]
yeah sakura is trying to fix it but its safe

**Comment by progsa98** [comment]
Thanks, can't download it, tried building from the source code, but I think that's still the sakura.1 version cause I get crashes with lithium installed

**Comment by Samalando** [comment]
if you click more info you can bypass it iirc, but I don't use windows so I cant help too much

**Comment by okibcn** [comment]
Yes, Windows detects it as a trojan and you have to create an exception if you want to use that version:
![image](https://github.com/maruohon/litematica/assets/22417711/c46a7cd9-44a3-416f-ac81-700743e09a72)

**Comment by LittlePerson101** [comment]
1.21 plz

**Comment by Barkells** [comment]
anyone know where to grab malilib for it?

**Comment by TacticianTilted** [comment]
Where do I get 1.21 Malib

**Comment by Cassy-Lee** [comment]
> Where do I get 1.21 Malib

https://github.com/sakura-ryoko/malilib/releases
